tests/integration/questionnaire/test_questionnaire_hub.py

```diff
@@ -120,3 +120,21 @@ def test_hub_section_url_when_hub_enabled_and_section_complete(self):
 
         # Then I should be redirected to the last completed question in the section
         self.assertEqualUrl('/questionnaire/accommodation-details-summary/')
+
+    def test_hub_inaccessible_if_sections_required_and_incomplete(self):
+        self.launchSurvey('test_hub_complete_sections')
```

Comment:
In the spirit of reducing integration tests, can we make this a functional test? Or is this needed for coverage?

Reply:
Coverage doesn't drop when I remove these, but the switch won't be tested. I'll add the alternative to the functional tests.